pkgs/development/tools/analysis/pev/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, openssl, fetchFromGitHub, ... }:
+stdenv.mkDerivation {
+  name = "pev-unstable-2018-07-22";
+  buildInputs = [ openssl.dev ];
+  phases = [ "unpackPhase" "buildPhase" "installPhase" ];
+  src = fetchFromGitHub {
+    owner = "merces";
+    repo = "pev";
+    rev = "aa4ef7f"; 
+    sha256 = "00a3g486343lhqcsf4vrdy5xif6v3cgcf2y8yp5b96x15c0wid36"; 
+    fetchSubmodules = true;
+  };
+
+  installPhase = ''
+    make install prefix=$out
+  '';
+
+  buildPhase = ''
+    make prefix=$out
```

Comment:
```
makeFlags = [  "prefix=$(out)" ];
installFlags = [  "prefix=$(out)" ];
```

instead of overriding the build phases.

Are both required or is `installFlags` sufficient?